Content.Shared/DragDrop/DropAttemptEvent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Robust.Shared.GameObjects;
+
+namespace Content.Shared.DragDrop
+{
+    public class DropAttemptEvent : CancellableEntityEventArgs
+    {
+        public DropAttemptEvent(IEntity entity)
+        {
+            Entity = entity;
+        }
+
+        public IEntity Entity { get; }
+    }
+
+    public static class DropAttemptExtensions
```

Comment:
if you're gonna do these extensions can they really not just be in one ActionBlockerExtensions class?

Reply:
Don't bother, we discussed this with PJB and agreed to remove the extensions and move them to entity system methods. I volunteered to change it.

Reply:
done